src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/GeneralConfiguredBrowseObject.xaml

```diff
@@ -0,0 +1,194 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--Copyright, Microsoft Corporation, All rights reserved.-->
+<Rule
+  Name="ConfiguredBrowseObject"
+  DisplayName="General"
+  PageTemplate="generic"
+  Description="General"
+  OverrideMode= "Replace"
+  xmlns="http://schemas.microsoft.com/build/2009/properties">
+
+  <Rule.Categories>
+    <Category Name="General" DisplayName="General" Description="General" />
+  </Rule.Categories>
+
+  <Rule.DataSource>
+    <DataSource Persistence="ProjectFile" HasConfigurationCondition="True" />
+  </Rule.DataSource>
+
+  <StringProperty Name="ApplicationIcon" DisplayName="Application Icon" Visible="True" />
```

Comment:
We'll also need to run @nguerrera's loc tool to generate the localized XAML rules as well. The tool is in https://github.com/nguerrera/xliff-converter with instruction to run in the readme. To make sure we are not doubling the number of strings, please run after you refactor the properties from the generalbrowseobject to this new rule.